app_dart/dev/branch_regexps.txt

```diff
@@ -1,2 +1,2 @@
-^v[0-9]+\.[0-9]+\.[0-9]+$
+^flutter-[0-9]\.[0-9]-candidate\.[0-9]+$
```

Comment:
Are you missing the "+" after the first and second [0-9]?

Reply:
Yes! Added.